tests/unit/python/foglamp/common/test_configuration_manager.py

```diff
@@ -2509,13 +2511,26 @@ def mock_coro():
         ("URL", "coap://host.co.in", True),
         ("URL", "coaps://host:6683", True),
         ("password", "not implemented", None),
-        ("X509 certificate", "not implemented", None)
+        ("X509 certificate", "not implemented", None),
+        ("northTask", "item", True),
+        ("northTask", "123", True)
     ])
     async def test__validate_type_value(self, item_type, item_val, result):
         storage_client_mock = MagicMock(spec=StorageClientAsync)
         c_mgr = ConfigurationManager(storage_client_mock)
         assert result == c_mgr._validate_type_value(item_type, item_val)
 
+    @pytest.mark.parametrize("input, expected", [
+        ('northTask', 'true'),
+        ('north', 'false'),
+        ('bool', 'false'),
+        ('boolean', 'true')
+    ])
+    async def test_valid_type(self, input, expected):
+        storage_client_mock = MagicMock(spec=StorageClientAsync)
+        c_mgr = ConfigurationManager(storage_client_mock)
+        assert expected == c_mgr._validate_type(input)
```

Comment:
There is NO _validate_type definition in configuration manager. Remove this test as it is NOT required